### DISCUSSION TITLE
这是一篇感谢贴，感谢@Mike777Khoo协助，成功退款了

### DISCUSSION THREAD

**Discussion by TangAssos** [getlantern/lantern-forum-cn#684] [Ideas]
之前申请退款有10于天，申请好后，@Mike777Khoo放话，14天还没到账，我就继续来找他，确实，没到账，我就又来找他了，他也确实兑现了承诺，这一波好感，我认他，蓝灯的前端技术，他控制不了，可是他很好的捍卫了蓝灯最后的尊严（收集用户诉求并解决），我就一普通消费者，买东西嘛，除了要东西好以外，售后人员这块太重要了，就像买车，我的车子用了12年，也是因为有负责任的售后，这12年来，我用车几乎没操心过；小问题多，不怕，只要有责任心强的售后，还是能赢回客户的好感；我本人用VPN跑代码和撸币，蓝灯10分钟就更换好几次IP，而且又断流，致使推特经常异常登录被锁号，撸币也经常断开，确实无法使用；未来，我会继续关注蓝灯论坛，前端做好了，@Mike777Khoo仍然是售后人员，我肯定还会冲，再次谢谢@Mike777Khoo协助退款，祝论坛里的各位大神早日发财，Mike777Khoo早日升官，大家江湖再见！
![退款成功](https://github.com/getlantern/lantern-forum-cn/assets/173048245/4b522773-e77e-46f9-b1be-72e57291f255)

**Comment by MaxMKMax** [comment]
@TangAssos 您好, 非常开心能收到您的感谢帖子! 非常感谢您对我的认可, 也对团队的认可! 除了我之外我们的客服部门 Nolan 也帮了很大的忙.

很荣幸能服务到您. 如果以后您还需要蓝灯的服务, 随时欢迎您再回来. 您也可以使用免费版, 等到您以后对我们更有信心了, 再回来购买. 

这一路真的非常感激您选择了蓝灯. 谢谢您. 希望以后再见到您. 以后有什么需要帮忙的, 请再找我. 我会在GitHub服务您. 如果您对我们的服务满意的话, 希望您能在IOS/ Android/ Google 应用商店给我们点个 5 星好评. 谢谢您! 

cc: @NolanRy
